spec/features/admin/admin_dashboard_spec.rb

```diff
@@ -32,25 +32,16 @@ def basic_auth!
     end
 
     scenario "viewing a pdf" do
-      create(:interview,
-             fee_agent_name: "Jessie Tester",
-             client_name: "Jane Doe",
-             attendee_names: "",
-             any_not_listed: "no",
-             any_away_from_home: "yes",
-             any_away_from_home_names: "James Bond")
+      create(:interview, fee_agent_name: "Jessie Tester")
       visit admin_root_path
 
       click_on "Download"
 
       temp_file = write_raw_pdf_to_temp_file(source: page.source)
       pdf_values = filled_in_values(temp_file.path)
 
+      # Minimal testing to make sure PDF isn't corrupted
       expect(pdf_values["fa_name"]).to include("Jessie Tester")
-      expect(pdf_values["applicant_name"]).to include("Jane Doe")
-      expect(pdf_values["any_not_listed"]).to include("no")
-      expect(pdf_values["any_away_from_home"]).to include("yes")
-      expect(pdf_values["any_away_from_home_reasons"]).to include("James Bond")
```

Comment:
Oh I see. I was adding all the tests here, when they should have been going in the decorator spec.

Reply:
yeah, it's easy to miss! realize it was something Whitney and I had built out but hadn't communicated. the decorator specs are pretty cool, I think, and should help us really easy-to-make PDF mistakes. maybe we can walk through them with Luigi on Monday